src/main/resources/database/changes/release-11/changelog.yml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+
+  - changeSet:
+      id: 11.-1
+      author: Samiul Islam
+      changes:
+        - sqlFile:
+            comment: Remove survey names
+            path: remove_survey_name.sql
+            relativeToChangelogFile: true
+            splitStatements: false
+
+  - changeSet:
+        id: 11.-2
```

Comment:
sorry to be fussy, but I'm really not a fan of -2 version id. Can we just make it 11.2 pls?